mock-thunderhead/src/main/java/com/sequenceiq/thunderhead/grpc/service/auth/MockUserManagementService.java

```diff
@@ -316,6 +318,9 @@
 
     private Optional<SshPublicKey> sshPublicKey;
 
+    @Value("${auth.mock.azure.disk.SSEWithCMk.enable}")
+    private boolean enableAzureDiskSSEWithCMk;
```

Comment:
Any reason for having a lowercase `k` at the end? Perhaps it would be better to have `CMK` all uppercase. (The other option is to use camelcase everywhere, i.e. `enableAzureDiskSseWithCmk`, but I have no strong preference either way.)

Reply:
It was a mistake. Should have been uppercase, updated to `enableAzureDiskSSEWithCMK`.